### ISSUE TITLE
TalkBack Doesn't Provide Feedback During Rearranging Icons on Home Screens

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#258]
```
What steps will reproduce the problem?
1. Go to the all apps screen.
2. Double-tap and hold an icon you want to move.
3. Try dragging to a desired location.
What is the expected output?
TalkBack should announce the position of an icon in relation to another icon to 
assist in moving that icon to a desired location on a home screen, or move an 
icon on top of another to create a folder.
What do you see instead?
TalkBack doesn't provide such feedback; the only spoken feedback is the 
announcement of home screen switching when moving an icon to the edge of the 
current home screen.
What version of the product are you using?
TalkBack 3.3.2
On what operating system?
4.2.2 Jellybean
Please provide any additional information below.
I find this to be the case on the Nexus 7, but can't test it on another tablet 
or phone to compare results.
```

Original issue reported on code.google.com by `mcigo...@gmail.com` on 3 May 2013 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
This appears to be an application accessibility issue and outside the scope of 
the eyes-free project.  I've reported this issue to the relevant team.
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 9:58
- Changed state: **Invalid**
